pull_request_template.md

```diff
@@ -8,13 +8,19 @@
 [Discuss the impact of your changes on the project. This might include effects on performance, new dependencies, or changes in behaviour.]
 
 ## Checklist
-- [ ] My code adheres to the coding and style guidelines of the project.
-- [ ] I have performed a self-review of my own code.
-- [ ] I have extended current unit tests suite.
-- [ ] I have made corresponding changes to the documentation.
-- [ ] My changes generate no new warnings or new Sonar issues
-- [ ] I have passed the PR linting stage
-- [ ] I contacted my SRE colleague to update the configuration (if any changes)
+- [ ] **[JIRA]** I have linked the JIRA ticket to the PR.
+- [ ] **[CODE]** My code adheres to the coding and style guidelines of the project.
+- [ ] **[REVIEW]** I have performed a self-review of my own code.
+- [ ] **[DOCS]** I have made corresponding changes to the documentation.
+- [ ] **[SONAR]** My changes generate no new warnings or new Sonar issues.
+- [ ] **[LINT]** I have passed the PR linting stage.
+- [ ] **[SRE]** I contacted my SRE colleague to update the configuration (if any changes)
+- [ ] **[LABELS]** I added the relevant labels to the PR.
+- [ ] **[DEPS]** I have added/updated external dependencies/packages.

```

Comment:
These two:

- [ ] **[LINT]** I have passed the PR linting stage.
- [ ] **[SRE]** I contacted my SRE colleague to update the configuration (if any changes)

existed previously.

The one about the labels is relevant because not everybody remembers to add labels and even more so the correct label.

Reply:
I was referencing line 19 only 
**[DEPS]** I have added/updated external dependencies/packages.

Reply:
In the webfront we want to have a dedicated checkbox related to dependencies so that whenever a dependency will change both QA and the DEV team will know when to trigger load testing.